ui/server/src/main/scala/pl/touk/nussknacker/ui/db/entity/ProcessVersionEntityFactory.scala

```diff
@@ -67,19 +59,4 @@ case class ProcessVersionEntityData(id: VersionId,
                                     json: Option[String],
                                     createDate: Timestamp,
                                     user: String,
-                                    modelVersion: Option[Int]) {
-
-  def graphProcess: GraphProcess = json match {
```

Comment:
Why you removed it? Having it here should make us closer to vision that json is always defined (not an Option type). We don't want to make it happen? In which case it is None in json?

Reply:
I've spoken with @mproch and we decided to remove it from Entity.